test/functions/whereClauses/whereClauseParamArgMismatch3.good

```diff
@@ -1,2 +1,2 @@
 whereClauseParamArgMismatch3.chpl:10: error: unresolved call 'foo(1.2)'
-whereClauseParamArgMismatch3.chpl:1: note: candidates are: foo(param x: uint(8))
+whereClauseParamArgMismatch3.chpl:1: note: candidates are: foo(param x: uint(64))
```

Comment:
I think you're right that it can go down distinct code paths.

For instance 

```chapel
proc foo(param x: int)     where (x % 2) == 1 { writeln(x, " is odd");    }
proc foo(param x: real(64))                   { writeln(x, " is a real"); }

foo(2.2)
```

compiles just fine on master today, but making x an` int(64)` in the first function causes it to break.